lib/screens/med_online/screens/doctors_card_screen.dart/doctors.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+
+import 'package:flutter_animation/screens/med_online/screens/doctors_card_screen.dart/domain/doctors_list.dart';
+import 'package:flutter_animation/screens/med_online/screens/doctors_card_screen.dart/widgets/animated_toggle.dart';
+import 'package:flutter_animation/screens/med_online/screens/doctors_card_screen.dart/widgets/doctor_app_bar.dart';
+import 'package:flutter_animation/screens/med_online/screens/doctors_card_screen.dart/widgets/doctors_list.dart';
+import 'package:flutter_animation/screens/med_online/screens/search_screen/widgets/search_med.dart';
+
+// ---Texts---
+
+// ---Parameters---
+const _kHeight = 20.0;
+const _kHeightPreferred = 56.0;
+
+class Doctors extends StatefulWidget {
+  const Doctors({Key? key}) : super(key: key);
+
+  @override
+  State<Doctors> createState() => _DoctorsState();
+}
+
+class _DoctorsState extends State<Doctors> {
+  final doctors = DoctorList().doctors;
+
+  @override
+  Widget build(BuildContext context) {
+    return GestureDetector(
+      onTap: () {
+        FocusScopeNode currentFocus = FocusScope.of(context);
+        if (!currentFocus.hasPrimaryFocus) {
+          currentFocus.unfocus();
+        }
+      },
+      child: Scaffold(
+        appBar: const PreferredSize(
+          preferredSize: Size.fromHeight(_kHeightPreferred),
+          child: DoctorAppBar(),
+        ),
+        body: Column(
+          children: [
+            const SearchMed(),
```

Comment:
<img width="407" alt="Screenshot 2022-07-14 at 14 10 11" src="https://user-images.githubusercontent.com/20111049/178969835-71eb64db-a50b-4e7c-a667-cf2412b8ae74.png">
Expected: user should see only one doctor
Actual: user see all doctors